README.md

```diff
@@ -51,6 +51,12 @@ $ sudo apt-get install build-essential gettext autoconf automake libproxy-dev li
 
 ```
 
+Under Fedora-based or CentOS-based distributions, this should install the requirements:
+
+```sh
+$ sudo yum-builddep openconnect
```

Comment:
The problem with this approach is that it assumes that the dependencies of this repository are identical to those of the version of OpenConnect in your distro.

The OpenConnect dependencies don't change *too* often, but they do occasionally. Can you list all the required packages explicitly, like I did for the Ubuntu/Debian version above?